### DISCUSSION TITLE
How to get Confidence level of NER predicted entities using spacy 3.1?

### DISCUSSION THREAD

**Discussion by karndeepsingh** [explosion/spaCy#10020] [Help: Coding & Implementations] [answered]
Hi, I have been training NER Transformer model using Spacy. I want the confidence level of each predicted entity , how this can be achieved?  Please, help me to  get confidence level as it is a key metrics to report in my organisation with predicted entities.

Thanks

**Comment by polm** [comment, marked as answer]
Answer is the same as when you asked in September. Train a spancat. 

https://github.com/explosion/spaCy/discussions/9189

**Reply by karndeepsingh** [reply]
Please help on how to train spancat with my Transformers model.  Does training spancat will help me to achieve the similar performance as transformer model? A proper guide could help me to understand the basics. Please help me.
Thanks

**Reply by svlandeg** [reply]
Duplicate of https://github.com/explosion/spaCy/discussions/9189 - locking this discussion to avoid duplicate threads.
